messaging/tests/test_services.py

```diff
@@ -13,7 +13,6 @@ class TestServices(unittest.TestCase):
   @parameterized.expand(service_list.keys())
   def test_services(self, s):
     service = service_list[s]
-    self.assertTrue(service.port > 8000)
```

Comment:
Why not leave this check? Import STARTING_PORT and compare with that as a sanity check.

Reply:
Also add a check the reserved port is not used.

Reply:
Right, reverted! I think I was brainwashed to optimize for lines haha, sorry. Is there anything else for this PR?